### DISCUSSION TITLE
Nushell as superuser

### DISCUSSION THREAD

**Discussion by nos1dot618** [nushell/nushell#10595] [Q&A] [answered]
Is there a way to launch nushell as superuser

**Comment by ghost** [comment, marked as answer]
`nu` is just a command - as long as the superuser has it in the path and access to it you can enter it by running `nu` (`sudo nu` / `doas nu` / within root shell `nu`).  
to make it the superusers default shell (i would not recommend this in case nushell has a extreme bug and can't be entered) you can use `chsh` (read the manpage - its a bit more complex)
